src/main/scala/beam/router/r5/NetworkCoordinator.scala

```diff
@@ -219,6 +231,8 @@ object NetworkCoordinator {
       val speeds = getSpeeds(highwayType)
       val capacities = getCapacities(highwayType)
       val lanes = getLanes(highwayType)
+      val alphas = getAlphas(highwayType)
+      val betas = getBetas(highwayType)
```

Comment:
Where do these alphas and betas go? It seems like they are not used anywhere.

Reply:
Good point, deleted.